docs/vocascan-frontend/configuration.md

```diff
@@ -3,8 +3,35 @@
 You can configure vocascan-frontend with the following options. For a instruction how to use them look at the individual
 [installation guides](vocascan-frontend/installation/installation.md).
 
+## Environment Variables
```

Comment:
can we keep this more general and then describe afterwards, that `REACT_APP_VOCASCAN_` + option is needed for build, `VOCASCAN_` + option is needed for docker start and then how config files work? Then there is no need to also mention this again and again.